qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -5,18 +5,20 @@
 
 from qcodes import Instrument, validators as vals
 
+log = logging.getLogger(__name__)
+
 
 class SignalHound_USB_SA124B(Instrument):
-    '''
+    """
     This is a direct port of the signal hound QTLab driver by Ramiro
-    Edited by Adriaan Rol
+    Edited by Adriaan Rol and others (use git blame for details)
 
     Status: Beta version.
         This driver is functional but not all features have been implemented.
 
     TODO:
         Add tracking generator mode
```

Comment:
can this line be reformulate from "TODO" to smth like : "Note that this driver does not support tracking generator mode". and then perhaps make a separate low-prio issue for "enabling tracking generator mode for ??? driver"?

Reply:
No, I think it's better kept as it is. Missing driver features are non-issues until they become issues, if you know what I mean.

Reply:
ok, agree. still, when I see "TODO", I understand it as developer-related thing, or a notion of a bug or deficiency. When the statement is formulated as smth like "Note that this driver does not support tracking generator mode", as a user I know that it hasn't been an issue so far and I do not expect the feature to be implemented UNLESS as a user I make an issue out of it on github.